Uno.Cupertino/Styles/Application/AnimationConstants.xaml

```diff
@@ -1,15 +1,14 @@
 ﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml">
 
-	<!--<CubicEase x:Key="MaterialEaseInOutFunction"
+	<!-- Based on animation length form https://github.com/dotnet/GraphicsControls/blob/master/src/GraphicsControls/Effects/RippleEffect.shared.cs -->
+	<CubicEase x:Key="MaterialEaseInOutFunction"
 			   EasingMode="EaseInOut" />
 
-	<CubicEase x:Key="MaterialEaseOutFunction"
-			   EasingMode="EaseOut" />
+	<Duration x:Key="MaterialAnimationDuration">0:0:0.35</Duration>
```

Comment:
Did we need these in Cupertino for something? I don't believe there is a ripple effect in iOS that we have to emulate

Reply:
We probably need other constants/functions here, so the file could be kept.
But it definitely needs to be hollowed out, to avoid confusion.

Reply:
removed